msdsl/model.py

```diff
@@ -36,11 +37,12 @@ def __init__(self, signal: Signal, n: Integral):
         self.n = n
 
 class MixedSignalModel:
-    def __init__(self, module_name, *ios, dt=None, build_dir='build'):
+    def __init__(self, module_name, *ios, dt=None, build_dir='build', real_type=RealType.FixedPoint):
```

Comment:
note the new optional argument **real_type**.  In most cases, there is no effect whether it is defined or not, but when tables or functions are used, it could change the generator result